### ISSUE TITLE
Top 10 Most dangerous places

### ISSUE THREAD

**Issue by trystant** [BOWiki/BOW#734]
I think that a top 10 Most Dangerous places based upon number of People of Color killed by police would be a good addition to the site. Question is, where to put the result?

**Comment by campgurus** [comment]
Good idea. I think we should standardize the results by population to report a per-capita number.  That way we make sure we are comparing apples to apples when we look at Tupelo, MO vs NYC. 

By the way, this may mean we need a Cities table, that tracks city name and population.  There may be a way to do this incorporating autocomplete using the Google Maps api.

I would put this report on the analytics page.

**Comment by rlgreen91** [comment]
This is a great idea.

**Comment by rlgreen91** [comment]
Wanted to bump this - I've been mulling over some things we might want to add for the search functionality - do you think this could be a good one @trystant @campgurus ?  I think we could, at least for the US, use public APIs to pull the population rather than storing it ourselves.  That would allow us to capture metro areas rather than each individual town/city as well.
